packages/react-devtools-shared/src/backend/StyleX/utils.js

```diff
@@ -90,13 +93,19 @@ function getPropertyValueForStyleName(styleName: string): string | null {
     const styleSheet = ((document.styleSheets[
       styleSheetIndex
     ]: any): CSSStyleSheet);
-    // $FlowFixMe Flow doesn't konw about these properties
-    const rules = styleSheet.rules || styleSheet.cssRules;
-    // $FlowFixMe `rules` is mixed
+    let rules: CSSRuleList | null = null;
+    // this might throw if CORS rules are enforced https://www.w3.org/TR/cssom-1/#the-cssstylesheet-interface
+    try {
+      rules = styleSheet.cssRules;
+    } catch (_e) {
+      continue;
+    }
+
     for (let ruleIndex = 0; ruleIndex < rules.length; ruleIndex++) {
-      // $FlowFixMe `rules` is mixed
-      const rule = rules[ruleIndex];
-      // $FlowFixMe Flow doesn't konw about these properties
+      if (!rules[ruleIndex] instanceof CSSStyleRule) {
```

Comment:
noob question, does `instanceof` have higher priority than `!`?

Reply:
no, it should be
```suggestion
      if (!(rules[ruleIndex] instanceof CSSStyleRule)) {
```

Reply:
You are right. Thank you!